packages/graphql-client/src/graphql-client.js

```diff
@@ -89,16 +89,16 @@ export class GraphQLClient {
     return await this.mutate(verifyEmailMutation, 'verifyEmail', { token });
   }
 
-  async resetPassword(token: string, newPassword: string): Promise<void> {
+  async resetPassword(token: string, newPassword: any): Promise<void> {
     return await this.mutate(resetPasswordMutation, 'resetPassword', { token, newPassword });
   }
 
-  async sendVerificationEmail(userId: string, email: string): Promise<void> {
-    return await this.mutate(sendVerificationEmailMutation, 'sendVerificationEmail', { userId, email });
+  async sendVerificationEmail(email: string): Promise<void> {
```

Comment:
The only use cases I can think of for the client to request verification email, are if you are enforcing login for verified only users, if you have non-verified status and the user wants to verify, or lastly, if an admin will run this request for user. In the first cases, the user is technically logged-in in some way, and in the later case the admin can identify users with whatever is available on the client (userId, email or username). This means that the input should probably be, optional complex type that can have either of those.